packages/gestalt/src/Contents.js

```diff
@@ -371,7 +371,7 @@ export default class Contents extends React.Component<Props, State> {
     mainDir: null,
   };
 
-  flyout = React.createRef();
+  flyout: {| current: null | HTMLDivElement |} = React.createRef();
```

Comment:
This is such a weird type - it would feel way more natural to do something like `flyout = React.createRef<HTMLDivElement>();` but obviously what do I know.